packages/agoric-cli/src/bin-agops.js

```diff
@@ -29,4 +32,25 @@ program.addCommand(await makePsmCommand(logger));
 program.addCommand(await makeReserveCommand(logger));
 program.addCommand(await makeVaultsCommand(logger));
 
-await program.parseAsync(process.argv);
+program.addCommand(
+  await makeInterCommand(
+    {
+      env: { ...process.env },
+      stdout: process.stdout,
+      stderr: process.stderr,
+      createCommand,
+      execFileSync,
+      clock: () => Date.now(),
+    },
+    { fetch },
+  ),
+);
+
+await program.parseAsync(process.argv).catch(err => {
+  if (err instanceof CommanderError) {
+    console.error(err.message);
+  } else {
+    console.error(err); // CRASH! show stack trace
+  }
+  process.exit(1);
+});
```

Comment:
since we do have top-level await, consider using regular try/catch
```suggestion
try {
  await program.parseAsync(process.argv);
} catch (err) {
  if (err instanceof CommanderError) {
    console.error(err.message);
  } else {
    console.error(err); // CRASH! show stack trace
  }
  process.exit(1);
};
```

Reply:
top-level `await` feels weird to me... but it clearly does exactly the same thing. What I want is for main programs to just export `main` and have the runtime call it, passing in all I/O access explicitly.

I guess that's why we're building endo. Meanwhile, when in Rome...